### PR TITLE
Add "Bug" Issue Template

### PR DESCRIPTION
**Pull request by KPrasch**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+**Describe the bug**
+A clear and concise description of what the bug is, and what you expected to happen.
+
+**To Reproduce**
+Steps to reproduce the behavior
+
+**Traceback or Screenshots (Optional) **
+If applicable, add the full error message / Traceback or upload screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (derekpierre) [inline .github/ISSUE_TEMPLATE/bug_report.md:19]:
```suggestion
**Configuration (please complete the following information):**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ - OS
+ - Nucypher Version [e.g. 2.1.0-beta.1]
```
Comment (derekpierre) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]:
```suggestion
 - Nucypher Version [e.g. 2.1.0-beta.1]
 - Python Version [e.g. 3.7]
 - Ethereum Node [eg. full, light, infura etc.]
```

**Comment by tuxxy** [comment]
Add an optional field for "other supporting details" for things that don't quite fit. :)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (KPrasch) [reply .github/ISSUE_TEMPLATE/bug_report.md]: :heavy_check_mark:

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (KPrasch) [reply .github/ISSUE_TEMPLATE/bug_report.md]: :heavy_check_mark:

**Pull request merged**